Missing dependency yajl

Looks like we are now missing a C dependency `yajl` for fast json parsing. This is my fault for breaking the develop integration tests. Most likely introduced as part of #147 

```
ackend_1         | [2018-04-24 22:49:46 +0000] [12] [INFO] Worker exiting (pid: 12)
backend_1         | Traceback (most recent call last):
backend_1         |   File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 209, in run
backend_1         |     self.sleep()
backend_1         |   File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 359, in sleep
backend_1         |     ready = select.select([self.PIPE[0]], [], [], 1.0)
backend_1         |   File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 244, in handle_chld
backend_1         |     self.reap_workers()
backend_1         |   File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 524, in reap_workers
backend_1         |     raise HaltServer(reason, self.WORKER_BOOT_ERROR)
backend_1         | gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
backend_1         |
backend_1         | During handling of the above exception, another exception occurred:
backend_1         |
backend_1         | Traceback (most recent call last):
backend_1         |   File "/usr/local/bin/gunicorn", line 11, in <module>
backend_1         |     sys.exit(run())
backend_1         |   File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 74, in run
backend_1         |     WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()
backend_1         |   File "/usr/local/lib/python3.6/site-packages/gunicorn/app/base.py", line 203, in run
backend_1         |     super(Application, self).run()
backend_1         |   File "/usr/local/lib/python3.6/site-packages/gunicorn/app/base.py", line 72, in run
backend_1         |     Arbiter(self).run()
backend_1         |   File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 231, in run
backend_1         |     self.halt(reason=inst.reason, exit_status=inst.exit_status)
backend_1         |   File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 344, in halt
backend_1         |     self.stop()
backend_1         |   File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 393, in stop
backend_1         |     time.sleep(0.1)
backend_1         |   File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 244, in handle_chld
backend_1         |     self.reap_workers()
backend_1         |   File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 524, in reap_workers
backend_1         |     raise HaltServer(reason, self.WORKER_BOOT_ERROR)
backend_1         | gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
backend_1         | [2018-04-24 22:49:46 +0000] [32] [ERROR] Exception in worker process
backend_1         | Traceback (most recent call last):
backend_1         |   File "/usr/local/lib/python3.6/site-packages/ijson/backends/__init__.py", line 38, in find_yajl_cffi
backend_1         |     yajl = ffi.dlopen('yajl')
backend_1         |   File "/usr/local/lib/python3.6/site-packages/cffi/api.py", line 141, in dlopen
backend_1         |     lib, function_cache = _make_ffi_library(self, name, flags)
backend_1         |   File "/usr/local/lib/python3.6/site-packages/cffi/api.py", line 802, in _make_ffi_library
backend_1         |     backendlib = _load_backend_lib(backend, libname, flags)
backend_1         |   File "/usr/local/lib/python3.6/site-packages/cffi/api.py", line 797, in _load_backend_lib
backend_1         |     raise OSError(msg)
backend_1         | OSError: ctypes.util.find_library() did not manage to locate a library called 'yajl'
backend_1         |
backend_1         | During handling of the above exception, another exception occurred:
backend_1         |
backend_1         | Traceback (most recent call last):
backend_1         |   File "/usr/local/lib/python3.6/site-packages/gunicorn/arbiter.py", line 578, in spawn_worker
backend_1         |     worker.init_process()
backend_1         |   File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 126, in init_process
backend_1         |     self.load_wsgi()
backend_1         |   File "/usr/local/lib/python3.6/site-packages/gunicorn/workers/base.py", line 135, in load_wsgi
backend_1         |     self.wsgi = self.app.wsgi()
backend_1         |   File "/usr/local/lib/python3.6/site-packages/gunicorn/app/base.py", line 67, in wsgi
backend_1         |     self.callable = self.load()
backend_1         |   File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 65, in load
backend_1         |     return self.load_wsgiapp()
backend_1         |   File "/usr/local/lib/python3.6/site-packages/gunicorn/app/wsgiapp.py", line 52, in load_wsgiapp
backend_1         |     return util.import_app(self.app_uri)
backend_1         |   File "/usr/local/lib/python3.6/site-packages/gunicorn/util.py", line 352, in import_app
backend_1         |     __import__(module)
backend_1         |   File "/var/www/entityservice/__init__.py", line 21, in <module>
backend_1         |     from entityservice.views import Status, Version, ProjectList, Project, ProjectClks, Run, RunStatus, RunResult, RunList
backend_1         |   File "/var/www/entityservice/views/__init__.py", line 2, in <module>
backend_1         |     from .project import *
backend_1         |   File "/var/www/entityservice/views/project.py", line 5, in <module>
backend_1         |     from ijson.backends import yajl2_cffi as ijson
backend_1         |   File "/usr/local/lib/python3.6/site-packages/ijson/backends/yajl2_cffi.py", line 65, in <module>
backend_1         |     yajl = backends.find_yajl_cffi(ffi, 2)
backend_1         |   File "/usr/local/lib/python3.6/site-packages/ijson/backends/__init__.py", line 40, in find_yajl_cffi
backend_1         |     raise YAJLImportError('Unable to load YAJL.')
backend_1         | ijson.backends.YAJLImportError: Unable to load YAJL.
```

--- a/backend/Dockerfile
+++ b/backend/Dockerfile
@@ -7,9 +7,10 @@ EXPOSE 8000
 # libffi-dev needed by anonlink
 # g++ needed by anonlink/cffi
 # gmp-dev, mpfr-dev, mpc1-dev needed by gmpy2
+# yajl, yajl-dev needed by ijson
 # shadow needed for groupadd and useradd (NB: shadow package only available in alpine >= 3.6)
 RUN (pip install --upgrade pip setuptools; \
-     apk add --no-cache postgresql-dev libffi-dev g++ gmp-dev mpfr-dev mpc1-dev shadow)
+     apk add --no-cache postgresql-dev libffi-dev g++ gmp-dev mpfr-dev mpc1-dev shadow yajl yajl-dev)
 
 ADD requirements.txt /var/www/requirements.txt
 

